src/app/management/budget/page.tsx

```
@@ -7,89 +7,191 @@ import { budgetApi } from '@/app/api/auth/api';
 import { useAuthStore } from '@/app/api/auth/useAuthStore';
 import { Button } from '@/components/ui/Button';
 import { Input } from '@/components/ui/Input_auth';
+import { useRouter } from 'next/navigation';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useRouter'. Consider to delete it or rename it.

'useRouter' is defined here:


(lint/suspicious/noRedeclare)